src/components/DropzoneArea.js

```diff
@@ -472,8 +474,10 @@ DropzoneArea.propTypes = {
     previewText: PropTypes.string,
     /** Shows styled Material-UI Snackbar when files are dropped, deleted or rejected. */
     showAlerts: PropTypes.bool,
+/** Shows styled Material-UI Snackbar when files are rejected or maximum files exceeded. */
+    showAlertsOnError: PropTypes.bool,
```

Comment:
Instead of adding a new prop, I'd like to add a backward-compatible behaviour to the existing `showAlerts` prop, like this:

- `showAlert` prop set to `true` continue to work as the current implementation
- `showAlert` prop set to `all` works as it is `true`
- `showAlert` prop set to `onlyErrors` works as your proposed `showAlertsOnError` prop

Reply:
Hi,

I've made the following changes:
- removed .vs/slnx.sqlite
- updated TS typings (I'm no TS expert, let me know if I did something wrong there)

I like your idea of re-using the showAlerts fields. I propose using an array of the messages (variants) to display. 

- showAlerts can be a boolean ("global" true or false for all alerts).
- showAlerts can be an array, with values 'error', 'info', 'success'... It will therefore be possible to select to view info and success (but not errors), or errors and info (but no success)...
   - showAlerts={['error']} for only errors
   - showAlerts={['error', 'info']} for both errors and info
   - showAlerts={['error', 'success', 'info']} is same as showAlerts={true}
   - showAlerts={[]} is same as showAlerts={false}